### DISCUSSION TITLE
Add bluetooth/ble support

### DISCUSSION THREAD

**Discussion by aayushrautela** [andyholmes/valent#598] [Feature Request]
As kde connect is having bluetooth backend, it would be great to have an option to pair with bluetooth as well.
https://invent.kde.org/network/kdeconnect-kde/-/commit/4dba3394a7afff1246933b66699a21d1f923633c

**Comment by andyholmes** [comment]
I doubt I'll get this anytime soon, but contributions would be welcome.
